doc/nrf/ug_bt_mesh.rst

```diff
@@ -0,0 +1,91 @@
+.. _ug_bt_mesh:
+
+Working with Bluetooth Mesh
+###########################
+
+The |NCS| provides support for developing applications using the Bluetooth Mesh protocol.
+The support is based on Zephyr's :ref:`bluetooth_mesh` implementation.
+
+.. _bt_mesh_ug_intro:
+
+Introduction
+************
+
+The `Bluetooth Mesh Profile Specification`_ is developed and published by the Bluetooth SIG.
+It allows one-to-one, one-to-many, and many-to-many communication, using
+the Bluetooth Low Energy protocol to exchange messages between the nodes on the network.
+All nodes in a Bluetooth Mesh network can communicate with each other, as long as there's a chain of nodes between them to relay the messages.
+The messages are encrypted with two layers of 128 bit AES-CCM encryption, allowing secure communication between thousands of devices.
+
+The end-user applications are implemented as a set of Mesh Models.
+The Bluetooth SIG defines some generic and reusable models in the `Mesh Model Specification`_, but vendors are free to define their own models.
+
+Read more about the Bluetooth Mesh specification on the Bluetooth SIG website:
+
+* `Bluetooth Mesh Glossary`_
+* `Bluetooth Mesh FAQ`_
+* `Bluetooth Mesh Study Guide`_
+
+.. _mesh_ug_supported features:
+
+Supported features
+******************
+
+The Bluetooth Mesh in |NCS| supports all mandatory features of the `Bluetooth Mesh Profile Specification`_ and the `Mesh Model Specification`_, as well as most of the optional features.
+
+The following features are supported by Zephyr's :ref:`zephyr:bluetooth_mesh`:
+
+* :ref:`zephyr:bluetooth_mesh_core`:
+
+  * Node role
+  * Relay feature
+  * Low power feature
+  * Friend feature
+
+* :ref:`zephyr:bluetooth_mesh_provisioning`:
+
+  * Provisionee role (GATT and Advertising Provisioning bearer)
+  * Provisioner role (only Advertising Provisioning bearer)
+
+* :ref:`GATT Proxy Server role <zephyr:bt_mesh_proxy>`
+* :ref:`zephyr:bluetooth_mesh_models`
+
+In addition to these features, |NCS| implements several models defined in the  `Mesh Model Specification`_, see :ref:`bt_mesh_models` for details.
+
+Configuration
+*************
+
+The Bluetooth Mesh support is controlled by :option:`CONFIG_BT_MESH`, which depends on the following configuration options:
+
+* :option:`CONFIG_BT`
+* :option:`CONFIG_BT_OBSERVER`
+* :option:`CONFIG_BT_PERIPHERAL`
```

Comment:
Here I would add a short information what the given option enables, for example:
`:option:`CONFIG_BT` - This option enables Bluetooth support.`
Just to give some minor info when skimming the text.

And yes, this might not be that important here, but is really required further below.